consoleApp/src/main/java/com/infoshare/Menu.java

```diff
@@ -35,6 +35,8 @@ private void handleUserChoice () {
                 switch (userChoice) {
                     case 1: {
                         System.out.println("Wprowadź nową ofertę wolontariatu");
+                        UserRegistration reg = new UserRegistration();
+                        reg.register();
```

Comment:
Można zrobić w jednej linijce, bo chyba nie będzie to później używane jeszcze?

Reply:
zrobione :+1: